src/Raven.Client/Documents/Operations/AI/Agents/AiAgentChatTrimmingConfiguration.cs

```diff
@@ -53,7 +54,7 @@ public AiAgentChatTrimmingConfiguration(AiAgentSummarizationByTokens tokensConfi
         /// Only one trimming strategy may be active at a time. Do not use this constructor in conjunction
         /// with the summarization-based constructor.
         /// </remarks>
-        public AiAgentChatTrimmingConfiguration(AiAgentTruncateChat truncateConfig, AiAgentHistoryConfiguration historyConfig = null)
+        internal AiAgentChatTrimmingConfiguration(AiAgentTruncateChat truncateConfig, AiAgentHistoryConfiguration historyConfig = null)
```

Comment:
This should be enabled by default, mind

Reply:
This one not, the other one with Summarization

Reply:
with history? or without?